File: GravityBikes/Models/Bike.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GravityBikes.Data.Models
{
    public class Bike
    {
        public int BikeId { get; set; }
        public int BikePrice { get; set; }
        public string BikeModel { get; set; }
        public byte BikeSize { get; set; }
        public byte BikeGender { get; set; }
        public string BikeDescription { get; set; }
        public string Url { get; set; }
        public string PublicId { get; set; }

    }
}

File: GravityBikes/ClientApp/src/app/_models/Bike.ts
export interface Bike {
    bikeId: number;
    bikePrice: number;
    bikeModel: string;
    bikeSize: BikeSizes;
    bikeGender: BikeGenders;
    bikeType: BikeTypes;
    url: string;
    description: string;


}
enum BikeSizes {
    s = 0,
    m = 1,
    l = 2,
    xl = 3
}

enum BikeGenders {
    male = 0,
    female = 1
}

export enum BikeTypes {
    downhill, freeride = 0,
    enduro = 1,
    forKids = 2
}
